src/components/Asset/AssetContent/MetaMain/NftTooltip.tsx

```diff
@@ -0,0 +1,78 @@
+import Copy from '@shared/atoms/Copy'
+import External from '@images/external.svg'
+import ExplorerLink from '@shared/ExplorerLink'
+import { NftMetadata } from '@utils/nft'
+import React, { ReactElement } from 'react'
+import styles from './NftTooltip.module.css'
+import explorerLinkStyles from '@shared/ExplorerLink/index.module.css'
+
+export default function NftTooltip({
+  nft,
+  address,
+  chainId,
+  isBlockscoutExplorer
+}: {
+  nft: NftMetadata
+  address: string
+  chainId: number
+  isBlockscoutExplorer: boolean
+}): ReactElement {
+  // Currently Ocean NFTs are not displayed correctly on OpenSea
+  // Code prepared to easily integrate this feature once this is fixed
+  //
+  // Supported OpeanSea networks:
+  // https://support.opensea.io/hc/en-us/articles/4404027708051-Which-blockchains-does-OpenSea-support-
+  const openseaNetworks = [1, 137, 8217]
```

Comment:
we should display this link just for the supported networks (eth, polygon , rinkeby)